### ISSUE TITLE
What is the value of textbox or searchbox?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria#1057]
All form fields have a value or status. For all form fields except for searchbox or textbox, the specification defines how value or status is specified (for checkbox with aria-checked, for listbox with option, etc.). I suggest to specify this for searchbox and textbox in the specification, because the HTML equivalent <code>input type=text</code> has an attribute for it (<code>value</code>), but <code>textarea</code> uses the text content as value. Thus it is not clear what applies to searchbox and textbox. It seems obvious that the text content should be used as value (and not e.g. <code>aria-valuetext</code>), but it is not defined yet.

**Comment by jnurthen** [comment]
Why not aria-valuetext that would echo HTML.
https://github.com/w3c/aria/issues/711

Nonetheless please continue discussion in that issue if needed.
